File: 5º Semestre/Linguagens de Programação/TP1/arvore.cs
using System;

namespace Arvore
{
    public Class Tree {  
        Node root; 
        public Tree() {  
            this.root = null;  
        }  
    }  

    public Class Node {
        Node esq;
        Node dir;
        int valor;

        public Node() {
            this.esq = null;
            this.dir = null;
            this.valor = 0;
        }
    }
    class Program
    {
        static void Main(string[] args)
        {
            Tree tree = new Tree();
            Console.WriteLine(tree.root);
        }
    }
}

File: 5º Semestre/Linguagens de Programação/TP1/arvore.py
# para criar uma árvore, é preciso criar a classe dos nodes da árvore 
# com variáveis que apontam para outros nodes além do dado que salva
class Node:

    def __init__(self, data):

        self.left = None
        self.right = None
        self.data = data

class Tree:
    def __init__(self, data):

        self.root = None
        self.data = data